paasta_tools/cli/cmds/logs.py

```diff
@@ -266,6 +272,7 @@ def paasta_log_line_passes_filter(
     components: Iterable[str],
     clusters: Sequence[str],
     instances: Iterable[str],
+    pods: Iterable[str] = None,
```

Comment:
That's a good point. After some digging, it looks like `paasta_log_line_passes_filter` is used only in "default" mode, and I think after #3048, we've made stdout/stderr to be the default component. Is it correct to assume that stdout/stderr is from app_output? And if that's the case, I don't think it would be a problem to add the condition.

Reply:
Decided to remove condition. Best if we made sure to not exclude any logs from non-app_output components.